README.md

```diff
@@ -35,3 +39,156 @@ understand.
 
 Finally, it passes these options to the correct mpirun executable of the
 selected MPI flavor.
+
```

Comment:
clean up the above?

also, add a short section "basic usage", where you explain how to use `mympirun`

one important thing to mention is how to separate options to `mympirun` from options to the application, see `--`

as soon as `mympirun` runs into an option it doesn't know, it assumes everything that follows is for the applicaiton, I think...